src/ir/module-utils.cpp

```diff
@@ -156,48 +161,23 @@ Counts getHeapTypeCounts(Module& wasm) {
         // is in flux, skip counting them to keep the type orderings in nominal
         // test outputs more similar to the orderings in the equirecursive
         // outputs. FIXME
-        counts.note(*super);
+        counts.include(*super);
```

Comment:
Is the comment above this still valid?

Reply:
Yep, previously we "skipped counting" by only calling `note` once, which actually set the count to 1. Now we set the count to 0, so the comment is more accurate than before.

Reply:
The supertype is encoded in the binary, though, isn't it? Why don't we want to bump the count here?

Reply:
It's what the comment explains; supertypes appear in the binary only in nominal mode but not in equirecursive mode, so don't count them in either mode to keep the outputs similar.